GiniVision/Classes/Core/AnalysisViewController.swift

```diff
@@ -194,6 +203,22 @@ import UIKit
         loadingIndicatorView.stopAnimating()
     }
     
+    /**
+     Shows an error when there was an error with either during the analysis or uploading a document
```

Comment:
Shows an error when there was an error with either <strike>during</strike> the analysis or **document upload**